doc/yarn/ray-skein.yaml

```diff
@@ -0,0 +1,68 @@
+name: ray
```

Comment:
Commands will need to be updated appropriately in this file as well. If possible, I recommend using sphinx's `literalinclude` (https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-literalinclude) in these cases so you don't need to duplicate code in both the docs and examples.